rust/src/nfs/nfs.rs

```diff
@@ -1668,14 +1668,14 @@ pub unsafe extern "C" fn rs_nfs_tx_get_alstate_progress(tx: *mut std::os::raw::c
                                                   -> std::os::raw::c_int
 {
     let tx = cast_pointer!(tx, NFSTransaction);
-    if direction == Direction::ToServer.into() && tx.request_done {
-        //SCLogNotice!("TOSERVER progress 1");
+    if (direction == Direction::ToServer.into() && tx.request_done) || (direction == Direction::ToClient.into() && tx.response_done) {
```

Comment:
Is this or part still needed? This seems to be redundant wrt the next block of `else if`..